synapse/cmds/cortex.py

```
@@ -192,3 +199,8 @@ def runCmdOpts(self, opts):
             # FIXME local typelib and datamodel
             disp = core.getPropRepr(form,valu)
             self.printf('%s - %s' % (disp,','.join(tags)))
+
+class NextSeq(s_cli.Cmd):
+    '''
+    Generate and display the next id in the named sequence.
+    '''
```

I'd throw a NotImplementedError exception here just to note its a WIP.